with-foundry/circuits/Verifier.toml

```diff
@@ -1,2 +1,2 @@
 return = "0x0000000000000000000000000000000000000000000000000000000000000003"
-y = "0x0000000000000000000000000000000000000000000000000000000000000003"
+y = "0x0000000000000000000000000000000000000000000000000000000000000003"
```

Comment:
Confused why we need this file again? This was used by Nargo before...

Reply:
ah you're right. I will delete it